generators/angular/templates/src/main/webapp/app/entities/_entityFolder_/update/_entityFile_-form.service.ts.ejs

```diff
@@ -200,12 +200,22 @@ _%>
   <%_ for (field of fields) {
         const { fieldName, fieldTypeBoolean, fieldTypeTimed, fieldTypeLocalDate } = field;
   _%>
-    <%_ if (field.id) { _%>
-      <%= fieldName %>: null,
-    <%_ } else if (fieldTypeBoolean) { _%>
-      <%= fieldName %>: false,
-    <%_ } else if (fieldTypeTimed) { _%>
-      <%= fieldName %>: currentTime,
+    <%_ if (field.fieldDefaultValueDefined) { _%>
+      <%_ if (field.fieldTypeCharSequence) { _%>
+        <%= fieldName %>: '<%= field.defaultValue %>',
+      <%_ } else if (fieldTypeTimed) { _%>
+        <%= fieldName %>: dayjs('<%= field.defaultValue %>'),	
+      <%_ } else { _%>
+        <%= fieldName %>: <%= field.defaultValue %>,
+      <%_ } _%>
+    <%_ } else { _%>
```

Comment:
```suggestion
    <%_ if (field.fieldTsDefaultValue) { _%>
        <%= fieldName %>: <%= field.fieldTsDefaultValue %>,
    <%_ } else { _%>
```

in the generator preparation:
```
mutateData(field, {
  fieldTsDefaultValue: ({ fieldTsDefaultValue , defaultValue, fieldTypeCharSequence, fieldTypeTimed }) => {
    if (fieldTsDefaultValue !== undefined) return String(fieldTsDefaultValue);
    if (defaultValue === undefined) return undefined;
    if (fieldTypeCharSequence) return `'${defaultValue}'`;
    if (fieldTypeTimed) return `dayjs('${defaultValue}')`;
    return String(defaultValue);
  },
});
```

This approach simplifies the template and adds support to 
```
@FieldTsDefaultValue('ts only default value')
```

Reply:
I'll also need to put the quotes around the String() produced values... in the example above, it would produce 

```
...
  fieldName: ts only default value,
...
```

Reply:
Done